samples/quickstart.js

```diff
@@ -17,36 +17,34 @@
 
 // [START firestore_quickstart]
 const Firestore = require('@google-cloud/firestore');
-
-const firestore = new Firestore({
-  projectId: 'YOUR_PROJECT_ID',
-  keyFilename: '/path/to/keyfile.json',
-});
-
-const document = firestore.doc('posts/intro-to-firestore');
-
-// Enter new data into the document.
-document.set({
-  title: 'Welcome to Firestore',
-  body: 'Hello World',
-}).then(() => {
-  // Document created successfully.
-});
-
-// Update an existing document.
-document.update({
-  body: 'My first Firestore app',
-}).then(() => {
-  // Document updated successfully.
-});
-
-// Read the document.
-document.get().then(doc => {
-  // Document read successfully.
-});
-
-// Delete the document.
-document.delete().then(() => {
-  // Document deleted successfully.
-});
-// [END firestore_quickstart]
\ No newline at end of file
+(async () => {
```

Comment:
Instead of the closure here, can you create a `main` method?

```js
async function main() {
  ...
}

main.catch(console.error);
```

By doing it this way, you can totally drop the try/catch as well :)

Reply:
Changes committed as suggested.